src/dc_tools.rs

```diff
@@ -661,43 +680,43 @@ mod tests {
         assert_eq!(grpid, None);
 
         // Should return None if grpid has a length which is not 11 or 16
-        let mid = "Gr.12345678.morerandom@domain.de";
-        let grpid = dc_extract_grpid_from_rfc724_mid(mid);
+        let mid = [GROUP_PREFIX, "12345678.morerandom@domain.de"].concat();
```

Comment:
hm, not sure what we really want to test here.
i'd say it is more important to make sure that things "from the wire" are parsed correctly - so an accidental change of GROUP_PREFIX should break the test.
therefore, maybe better leave the test using a hardcoded string as before.
of course, we can add additional tests using GROUP_PREFIX.